sdk/cosmos/azure-cosmos/test/test_crud_async.py

```diff
@@ -1357,17 +1357,17 @@ async def __SetupEntities(client):
                                             "Session",
                                             connection_policy=CRUDTests.connectionPolicy)
         # setup entities
-        entities = __SetupEntities(client)
+        entities = await __SetupEntities(client)
         resource_tokens = {"dbs/" + entities['db'].id + "/colls/" + entities['coll'].id:
                                entities['permissionOnColl'].properties['_token']}
         col_client = cosmos_client.CosmosClient(
             CRUDTests.host, resource_tokens, "Session", connection_policy=CRUDTests.connectionPolicy)
         db = entities['db']
 
-        old_client_connection = db.client_connection
-        db.client_connection = col_client.client_connection
+        old_client_connection = await db.client_connection
+        db.client_connection = await col_client.client_connection
         # 1. Success-- Use Col Permission to Read
-        success_coll = db.get_container_client(container=entities['coll'])
+        success_coll = await db.get_container_client(container=entities['coll'])
```

Comment:
doesn't need await - get_client() methods only make an object instance, not any network calls